torch/fx/experimental/graph_manipulation.py

```diff
@@ -255,9 +248,31 @@ def serialize_module(fx_module: GraphModule, weights: Dict, name_prefix="") -> D
 
         node_rep["op_code"] = node.op
         node_rep["name"] = node.name
-        node_rep["args"] = map_arg(
-            node.args, lambda arg: {"is_node": True, "name": str(arg)}
-        )
+
+        if node.op == "output":
+            def get_output_info(arg: Node) -> Argument:
+                shape, dtype = get_shape_and_dtype(arg)
+                return {
+                    "is_node": True,
+                    "name": str(arg),
+                    "shape": serialize_shape(shape),
+                    "dtype": str(dtype),
+                }
+
+            node_rep["args"] = map_arg(
+                node.args,
+                get_output_info,
+            )
+
+            # If there're multiple outputs then node_rep["args"][0] will be a tuple.
+            # In this case we want to unpack the tuple.
+            if isinstance(node_rep["args"][0], tuple):
+                node_rep["args"] = node_rep["args"][0]
+        else:
+            node_rep["args"] = map_arg(
+                node.args, lambda arg: {"is_node": True, "name": str(arg)}
+            )
+
```

Comment:
Not sure if these changes should be part of this PR?

Reply:
Those changes should belong to another PR. Sorry about this, I'll re-export and land that one first.

Reply:
Looks like it got fixed. Please take a look when you have time @jamesr66a, thanks!